WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift

```diff
@@ -103,11 +113,35 @@ private extension TabbedTotalsCell {
         siteStatsDetailsDelegate?.tabbedTotalsCellUpdated?()
     }
 
+    func toggleNoResults() {
+        noResultsViewController.removeFromView()
+
+        let showNoResults = tabsData[filterTabBar.selectedIndex].dataRows.isEmpty && !limitRowsDisplayed
+        noResultsView.isHidden = !showNoResults
+
+        guard showNoResults,
+            let superview = superview else {
+                return
+        }
+
+        noResultsViewController.view.frame = noResultsView.frame
+        noResultsViewController.view.frame.origin.y = 0
+        noResultsViewController.view.frame.size.height = superview.frame.height - filterTabBar.frame.height
+        noResultsView.addSubview(noResultsViewController.view)
```

Comment:
I see where we do the view controller containment dance for removal on line 117. Do we need to do the opposite on or about line 130?

Reply:
I've struck through my comment above, as it became clear that we this class (`TabbedTotalsCell`) is not a view controller.

Reply:
I misunderstood that `TabbedTotalsCell` is sort-of overloaded. 

I see now that it's used to help render a card on the parent view (`SiteStatsInsightsTableViewController`), but on the child view (`SiteStatsDetailTableViewController`), it's _almost_ like a root view.

Carry on. 😄